sdk/ai/Azure.AI.Projects/tests/AgentClientTests.cs

```diff
@@ -49,8 +49,10 @@ public enum VecrorStoreTestType
         #endregion
 
         [RecordedTest]
-        [TestCase(ArgumentType.Metadata)]
-        [TestCase(ArgumentType.Bytes)]
+        //Failing in CI due to no playback found
```

Comment:
Can we please open an issue to track or just rerecord the tests?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/48725